lib/Controllers/create.js

```diff
@@ -17,7 +17,29 @@ Create.prototype.plurality = 'plural';
 Create.prototype.write = function(req, res, context) {
   context.attributes = _.extend(context.attributes, req.body);
 
-  var self = this;
+  var self = this,
```

Comment:
I work on it. The resource will calculate the information of associations at creation.
It will be good ;)